api/wikis/serializers.py

```diff
@@ -0,0 +1,71 @@
+from rest_framework import serializers as ser
+
+from api.base.serializers import JSONAPISerializer, IDField, TypeField, Link, LinksField, RelationshipField
+from api.base.utils import absolute_reverse
+
+
+class WikiSerializer(JSONAPISerializer):
+
+    filterable_fields = frozenset([
+        'name',
+        'date_modified'
+    ])
+
+    id = IDField(source='_id', read_only=True)
+    type = TypeField()
+    name = ser.CharField(source='page_name')
+    path = ser.SerializerMethodField()
+    materialized = ser.SerializerMethodField(method_name='get_path')
+    date_modified = ser.DateTimeField(source='date')
+    content_type = ser.SerializerMethodField()
+    extra = ser.SerializerMethodField(read_only=True, help_text='Additional metadata about this wiki')
```

Comment:
Also no `size` field. I forget if we talked about skipping that one, though shouldn't be hard to implement I think.